src/linear_solver_logs/solve_log_ma.jl

```diff
@@ -183,17 +184,18 @@ function log_update!(
     if !log.true_res && iter > 0
         # Compute the current residual to second power to align with theory
         # Check if it is one dimensional or block sampling method
-        res::Float64 = eltype(samp[1]) <: Int64 || size(samp[1],2) != 1 ? 
-            log.resid_norm(samp[3])^2 : log.resid_norm(dot(samp[1], x) - samp[2])^2 
+        res::Float64 = log.dist_info.scaling * (eltype(samp[1]) <: Int64 || size(samp[1],2) != 1 ? 
+                                                log.resid_norm(samp[3])^2 : log.resid_norm(dot(samp[1], x) - samp[2])^2) 
```

Comment:
Why is there so much white space added in?

Reply:
At the time I thought it was more clear to have the if else statement within the parentheses aligned. I know see that is not the case and I have removed it.